lib/Froxlor/Cron/Http/Apache.php

```diff
@@ -168,7 +168,7 @@ public function createIpPort()
 				$without_vhost = $this->virtualhosts_data[$vhosts_filename];
 				$close_vhost = true;
 
-				$this->virtualhosts_data[$vhosts_filename] .= '<VirtualHost ' . $ipport . '>' . "\n";
+				$this->virtualhosts_data[$vhosts_filename] .= '<VirtualHost *:' . $row_ipsandports['port'] . '>' . "\n";
```

Comment:
Ehm, no...this has nothing to do with quotes in DNS/SPF entries...

Reply:
This change wasn't supposed to go here, i did the pull request before doing this commit to my clone of the repository, i'm looking on how do i edit the pull request to remove it.

Reply:
Ok, i was able to change the pull request and remove these changes, as they were personal to my installation and didn't want to go into it.